src/components/SidebarSearch/SidebarSearch.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import cx from 'classnames';
+
+import Button from 'Button';
+
+type SidebarSearchProps = {
+  className?: string;
+  id: string;
+  label?: string;
+  onChange: () => void;
+  onClick: () => void;
+  onKeyDown: () => void;
+  onKeyUp: () => void;
+  value: string;
+};
+
+export const SidebarSearch = ({
+  className,
+  id,
+  label,
+  onChange,
+  onClick,
+  onKeyDown,
+  onKeyUp,
+  value
+}: SidebarSearchProps) => {
+  const classNames = cx('cc-sidebar-search', {
+    [className]: className
+  });
+
+  return (
+    <div className={classNames}>
+      <label htmlFor={`search-${id}`} className="cc-sidebar-search__label">
+        {label || 'Search by keyword'}
```

Comment:
Rather than using a conditional, I would suggest setting "Search by keyword" as the default value for the `label` prop.

```
SidebarSearch = ({
  ...otherProps,
  label = 'Search by keyword'
}) => {
```

That way it becomes clearer, earlier that there is a default value for label if you do not pass one.